app/Containers/AppSection/Product/UI/API/Controllers/Controller.php

```diff
@@ -85,8 +85,17 @@ public function updateProduct(UpdateProductRequest $request): array
      */
     public function deleteProduct(DeleteProductRequest $request): JsonResponse
     {
-        app(DeleteProductAction::class)->run($request);
+        $response = app(DeleteProductAction::class)->run($request);
 
-        return $this->noContent();
+        if ($response instanceof JsonResponse){
```

Comment:
return $this->deleted([
            'message' => 'delete successfully',
            'status' => 500,
        ]);